Content.Server/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -169,6 +187,32 @@ void InitializeDamageType(DamageType damageType)
                 Thresholds.Add(damageType, new List<DamageThreshold>());
             }
         }
+
+        /// <summary>
+        ///     Completely removes all damage from the DamageableComponent (heals the mob).
+        /// </summary>
+        [Verb]
+        private sealed class RejuvenateVerb : Verb<DamageableComponent>
+        {
+            protected override string GetText(IEntity user, DamageableComponent component) => "Rejuvenate";
+
+            protected override VerbVisibility GetVisibility(IEntity user, DamageableComponent component)
+            {
+                var groupController = IoCManager.Resolve<IConGroupController>();
+
+                if (user.TryGetComponent<IActorComponent>(out var player))
+                {
+                    if (groupController.CanCommand(player.playerSession, "rejuvenate"))
+                        return VerbVisibility.Visible;
+                }
+                return VerbVisibility.Invisible;
+            }
+
+            protected override void Activate(IEntity user, DamageableComponent component)
+            {
+                component.HealAllDamage();
```

Comment:
You should check whether the user still has the command in here, as the verb system does not treat `GetVisibility` for access control.

Reply:
Done.